Ical.Net/DataTypes/CalDateTime.cs

```diff
@@ -373,67 +316,144 @@ public DateTime Value
     public string? TimeZoneName => TzId;
 
     /// <summary>
-    /// Gets the year that applies to the <see cref="Value"/>.
+    /// Gets the year.
+    /// </summary>
+    public int Year => _localDate.Year;
+
+    /// <summary>
+    /// Gets the month.
+    /// </summary>
+    public int Month => _localDate.Month;
+
+    /// <summary>
+    /// Gets the day.
     /// </summary>
-    public int Year => Value.Year;
+    public int Day => _localDate.Day;
```

Comment:
Do the wrapped Day, Month, Year, Hour etc. properties bring a benefit to users?
Exposing `DateTime` could be sufficient?